patches/babel-plugin-emotion+10.2.2.patch

```diff
@@ -0,0 +1,112 @@
+diff --git a/node_modules/babel-plugin-emotion/dist/babel-plugin-emotion.cjs.dev.js b/node_modules/babel-plugin-emotion/dist/babel-plugin-emotion.cjs.dev.js
```

Comment:
Can you share more about this patch?

Reply:
This basically makes the plugin to handle styled imports from your theming package - instead of the `@emotion/styled` pkg.

Most importantly this makes "components as selectors" to work. In v11 we have a babel plugin option to tell us which imports are just wrapper/proxies to our APIs so that could be just configured in the v11. But since you are stuck at v10 i had to hack this with patch-package in all of your repos

Reply:
Aha. Thanks for explaining! 👍